README.md

```diff
@@ -1,20 +1,81 @@
+## Table of Contents
 
-We maintain a demo site at [demo.dataverse.org][] which you are welcome to use for testing and evaluating Dataverse.
+1. [❓ What is Dataverse?](#what-is-dataverse)
+2. [✔ Try Dataverse](#try-dataverse)
+3. [🌐 Our Web Presence](#our-web-presence)
+4. [📥 Installation](#installation)
+5. [🏘 Community and Support](#community-and-support)
+6. [🧑‍💻️ Contributing](#contributing)
+7. [⚖️ Legal Information](#legal-informations)
 
-To install Dataverse, please see our [Installation Guide][] which will prompt you to download our [latest release][]. Docker users should consult the [Container Guide][].
+<a name="what-is-dataverse"></a>
 
-To discuss Dataverse with the community, please join our [mailing list][], participate in a [community call][], chat with us at [chat.dataverse.org][], or attend our annual [Dataverse Community Meeting][].
+## ❓  What is Dataverse?
 
-We love contributors! Please see our [Contributing Guide][] for ways you can help.
+Welcome to Dataverse®, the [open source][] software platform designed for sharing, finding, citing, and preserving research data. Developed by the Dataverse team at the [Institute for Quantitative Social Science](https://iq.harvard.edu/) and the [Dataverse community][], our platform is built to support researchers and institutions in sharing, finding, citing, and preserving research their data effectively.
+
+<a name="try-dataverse"></a>
+
+## ✔  Try Dataverse
+
+We invite you to explore our demo site at [demo.dataverse.org][]. This site is ideal for testing and evaluating Dataverse in a risk-free environment.
+
+<a name="our-web-presence"></a>
+
+## 🌐  Our Web Presence
+
+Visit [dataverse.org][], our home on the web, for a comprehensive overview of Dataverse. Here, you will find:
+
+- An interactive map showcasing Dataverse installations worldwide.
+- A detailed list of [features][].
+- Information on [integrations][] that have been made possible through our [REST APIs][].
+- Our [project board][] and development [roadmap][].
+- News, events, and more.
+
+<a name="installation"></a>
+
+## 📥 Installation
+
+Ready to get started? Follow our [Installation Guide][] to download and install the latest release of Dataverse.
+
+If you are using Docker, please refer to our [Container Guide][] for detailed instructions.
+
+<a name="community-and-support"></a>
+
+## 🏘 Community and Support
+
+Engage with the vibrant Dataverse community through various channels:
+
+- **[Mailing List][]**: Join the conversation on our [mailing list][].
+- **[Community Calls][]**: Participate in our regular [community calls][] to discuss new features, ask questions, and share your experiences.
+- **[Chat][]**: Connect with us and other users in real-time at [chat.dataverse.org][].
```

Comment:
Hmm, or we could say "at https://dataverse.zulipchat.com". I dunno. Either way. I feel pretty committed to Zulip at this point. 🤷